.circleci/config.yml

```diff
@@ -92,7 +88,8 @@ jobs:
               cd pangeo
               helm dependency update
               cd -
-              helm upgrade --debug testhub --wait --timeout=1200 --install pangeo --namespace=test -f ./tests/minikube-config.yaml
+              kubectl create namespace test
+              helm upgrade --debug testhub --wait --timeout=1200s --install pangeo --namespace=test -f ./tests/minikube-config.yaml
```

Comment:
I learned recently from @scottyhq in https://github.com/yuvipanda/hubploy/issues/55 about the `--atomic` flag, which I think think should be used as a default. With it, `--wait` is implied.